storey/aggregations.py

```diff
@@ -584,21 +628,30 @@ def calculate_features(self, timestamp):
         if current_time_bucket_index < 0:
             return result
 
+        all_windows = []
```

Comment:
Better pass what windows you want to calculate the features for, because sometimes you only need the explicit and sometimes only the hidden and sometimes as mentioned in another comment you want specific hidden windows.